### ISSUE TITLE
Skip SMS Payment Option Screen

### ISSUE THREAD

**Issue by vladaman** [sumup/sumup-android-sdk#127]
New version shows option where merchant can select between Terminal and SMS Paymemt. Is there any option to remove this screen so merchant does not have to select this every time, since they don't use SMS Payment?

**Comment by jmar83** [comment]
I have the same question...

Maybe an employee of the development team can give us a a feedback about our question?

Thanks in advance!

**Comment by Mike-Lopez** [comment]
Hello,

Unfortunately it has to be disabled manually by support. If you email integration@sumup.com with the email address registered with SumUp. We can manually disable that for you.

Best

**Comment by jmar83** [comment]
OK thx. I already asked how to disable that screen via my business e-mail-account. Now i'm still waiting for the answer...

**Comment by Mike-Lopez** [comment]
Hi, 

unfortunately you will need to contact integration@sumup.com to deactivate your SMS payment.

Best

**Comment by jmar83** [comment]
ok thx, will ask about it..:-)

**Comment by fred4o** [comment]
Yes you need to disable from merchant acount in web or SumUp App and then screen disapear

**Comment by superafroman** [comment]
This option has been removed in the latest SDK so this issue should be closed.
